teams/vsa/design/vsa-research-process.md

```diff
@@ -78,6 +78,10 @@ We have to be careful with -- and if possible avoid using -- recordings because
    * Type “observer instructions” in the channel at the beginning of your day of sessions to give them tips
    * [Observer instructions can also be found here](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/research/during-research/howto-observer-instructions.md)
    * At the end of your session, take a look at the feedback backchannel and see if anyone brought up questions that you can address with your participant
+* A [Mural template](https://app.mural.co/template/0e63994b-bce9-4b53-8e37-b5293a2b4dac/20381d25-a5d5-4457-9893-eeb2e9cb6ff4) is available to help observers take notes during your sessions.
+  * The template has a section for each participant, and within the participant section, you can create a section for each task. 
+  * Share the mural board with your team, a day in advance of starting any sessions. Ask them to write any observations they have on the mural board.  Usually there are maybe 1-2 people who will be actively writing things, and that’s okay.  
```

Comment:
Maybe add here that we don't expect the moderator or the official "note taker" to be using the Mural board, but it's for any other observers that might want to put in notes during the sessions